tests/data/hpl_results.log

```diff
@@ -0,0 +1,1063 @@
+================================================================================
```

Comment:
Also has license concern.

Reply:
This is console std output from an HPL runs. Not sure what to do with this short of removing the top portion ... This is used for the unit-testing. I could truncate the file to what's absolutely necessary to pass the unit-test. Let me know what you would like me to do.

Reply:
I think it's OK for output data check